### ISSUE TITLE
Zenodo and DOI

### ISSUE THREAD

**Issue by NeuroShepherd** [chillemille/RIllustrations#3]

**Comment by chillemille** [comment]
I've just finished registering the repo in Zenodo and now we have a DOI :)

**Comment by NeuroShepherd** [comment]
Super! Could you add the DOI and Zenodo link to this issue? We should be good to close it after that.

**Comment by chillemille** [comment]
yep, sure: the DOI is "10.5281/zenodo.8185618" and the link is "https://doi.org/10.5281/zenodo.8185618". 

Just to make sure I proceeded correctly: I stuck with the main branch (default) when creating the release on Github since you said we're going to merge the milena-dev-book branch with main when we're ready (I have to add the references this morning and switch the order of the .Rmd files and then we should be ready to go)

**Comment by NeuroShepherd** [comment]
Yep, thats's correct! I think we'll technically need to create an updated DOI after we release a draft, but I'm not *super* familiar with the process of using DOIs.
